tests/DCPS/Thrasher/ParticipantTask.cpp

```diff
@@ -39,9 +40,9 @@ ParticipantTask::~ParticipantTask()
 int
 ParticipantTask::svc()
 {
-  try {
-    ACE_DEBUG((LM_INFO, ACE_TEXT("(%P|%t)    -> PUBLISHER STARTED\n")));
-
+  std::string pfx("%T pub");
```

Comment:
I disagree with the approach of a dynamic format string.  Use a static format for all logging statements and inject the appropriate variables.

Reply:
See the comment below about the format string.